shell/babel.config.js

```diff
@@ -0,0 +1,91 @@
+const overwriteMap = {
+  table: true,
+  select: true,
+  tag: true,
+  'range-picker': true,
+};
+
+const overwriteCssMap = {
+  table: 'antd/es/table/style',
+  select: 'antd/es/select/style',
```

Comment:
`table` & `select` seems is using antd style and don't need ignore. Still required to have these two properties here?

Reply:
Yes, it's useless, removed.